plottr/apps/inspectr.py

```diff
@@ -638,6 +648,27 @@ def starSelectedRun(self) -> None:
     def crossSelectedRun(self) -> None:
         self.tagSelectedRun('cross')
 
+    @Slot()
+    def exportSelectedRun(self) -> None:
+        """
+        Open a file dialog that allows selecting a folder and file name.
+        """
+        curdir = os.getcwd()
+        runId = int(self.runList.selectedItems()[0].text(0))
+        default_name = curdir+"/dataID_"+str(runId)+".csv"
```

Comment:
i'd highly recommend to use dataset's GUID in the file name. For an example you can see how qcodes' dataset export feature is implemented see here https://microsoft.github.io/Qcodes/examples/DataSet/Exporting-data-to-other-file-formats.html